worlds/tits_the_3rd/__init__.py

```diff
@@ -124,14 +124,20 @@ def setup_characters(self, itempool: Counter[str]) -> Counter[str]:
             for character in character_list:
                 if character in default_character_to_location:
                     location_name = default_character_to_location[character]
+                    if self.options.name_spoiler_option:
+                        real_location_name = scrub_spoiler_data(location_name)
+                    else:
+                        real_location_name = location_name
```

Comment:
It seems like we do this block a lot. Lets make a wrapper for this? So we can just do:
`self.multiworld.get_location(resolve_name(location_name), ...)`